src/assets/scss/_give.scss

```diff
@@ -523,6 +523,137 @@ hr.horizontal-divider {
   font-weight: 700;
 }
 
+.button-group {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+
+  .help-block {
+    padding: 10px;
+    border: 2px solid #f9b625;
+    border-radius: 2px;
+    background-color: lighten($colorCru-gold, 10%);
+    color: darken($colorCru-gold, 30%);
+  }
+
+  input[type="radio"] {
+    position: absolute;
+    cursor: pointer;
+    opacity: 0;
+  }
+
+  label {
+    display: flex;
+    flex: 1;
+    margin: 0;
+    cursor: pointer;
+  
+    .amount-box {
+      min-width: 50px;
+      align-content: center;
+      border-radius: 2px 0 0 2px;
+      margin: 0;
+      aspect-ratio: 1 / 1;
+      background-color: $colorCru-border;
+      color: inherit;
+      text-align: center;
+      transition: all 0.3s ease;
+    }
+
+    .amount-description {
+      width: 100%;
+      align-content: center;
+      padding: 5px 10px;
+      margin: 0;
+      background-color: $colorGrayscale-pebble;
+      font-weight: normal;
+    }
+  }
+
+  input[type="radio"]:hover + label {
+    .amount-description {
+      background-color: $colorGrayscale-pebble;
+    }
+  }
+
+  input[type="radio"]:focus + label {
```

Comment:
We had also talked about adding some kind of focus ring to the active radio button. Maybe @rguinee has thoughts on the look and feel, but I'm thinking something like this (just with rounded borders on the left side).

```scss
input[type="radio"]:focus.active + label {
  border: 1px solid #3eb1c8;
}
```